scss/main.scss

```diff
@@ -251,6 +250,20 @@ textarea {
   }
 }
 
+.btn-signout {
+  background: transparent;
```

Comment:
This shouldn't be needed? See my earlier comment regarding sticking with existing classes `btn-link` and `inline-link`.